addons/osio-addon/src/main/java/io/fabric8/launcher/osio/steps/booster/GolangBooster.java

```diff
@@ -0,0 +1,264 @@
+package io.fabric8.launcher.osio.steps.booster;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.math.NumberUtils;
+import org.codehaus.plexus.util.FileUtils;
+
+import io.fabric8.launcher.core.api.Projectile;
+import io.fabric8.launcher.core.spi.ProjectileEnricher;
+import io.fabric8.launcher.osio.projectiles.OsioLaunchProjectile;
+import io.fabric8.launcher.service.git.api.GitService;
+
+/**
+ * Customizes golan booster files.
+ *
+ */
+@ApplicationScoped
+public class GolangBooster implements ProjectileEnricher {
+
+    @Inject
+    GitService gitService;
+
+    private static final Logger log = Logger.getLogger(GolangBooster.class.getName());
+    private static final String GO_FILE_EXTENSION = ".go";
+    private static final String ENVIRONMENT = "environment";
+    private static final String ASSEMBLE = "assemble";
+    private static final String MAKEFILE = "Makefile";
+    private static final String GIT = "git";
+    private static final String URL = "url";
+    private static final String SOURCE = "source";
+    private static final String START_IMPORT = "import (";
+    private static final String END_IMPORT = ")";
+    private static final String NEW_LINE = "\n";
+    private static final String NO_FILE_EXTENSION = "";
+    private static final String SEPARATOR = "/";
+
+    private Path projectLocation;
+    private Map<String, Object> boosterData;
```

Comment:
Why the need to store these attributes since they are already available in the `accept` method parameter?

Reply:
Hi George, thanks for your review. 

Sorry, not done cleaning up, i think it would be better if i maybe pinged you when I'm ready to have it reviewed again?

This list existed prior because i was returning a list to be returned to gitSteps. However, since you delivered the enricher and i no longer utilize anything from gitSteps this doesn't make sense.

I definitely do not want to waste your time and will definitely let you know as soon as I am ready!! 

Thanks again!

Reply:
I should also mention, the ProjectileEnricher you delivered helped clean up a lot of stuff :-) thanks for delivering that so promptly. This became a lot cleaner afterwards - really appreciate it!